server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/managesieve/ManageSieveMailet.java

```diff
@@ -135,7 +133,7 @@ public void service(Mail mail) throws MessagingException {
 
         // Update the Session for the current mail and execute
         SettableSession session = new SettableSession();
-        if (mail.getAttribute(SMTP_AUTH_USER_ATTRIBUTE_NAME) != null) {
+        if (mail.getAttribute(Mail.SMTP_AUTH_USER_ATTRIBUTE_NAME) != null) {
```

Comment:
Use the constant introduced for RELAYING email here.

Hence the code will be more robust. If enabled, a JMAP user will be allowed to interact with the ManageSieveMailet, and not just SMTP users. This is desirable, and do not cost much. You can do i in a separate commit if you want.